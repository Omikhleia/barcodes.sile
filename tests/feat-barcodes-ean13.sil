\begin[papersize=a6]{document}
\neverindent
\nofolios
\use[module=packages.barcodes.ean13]
% We use the "undocumented" showdigits option to remove the human readable interpretation,
% so that not to bother with its font. These tests are focussed on checking the barcode
% vs. some checked results, so we are just interested in the bars.

% GS1 example (figure 5.2.2.1-2)
\ean13[scale=SC0, showdigits=false, code=9501101531000]

% GS1 example (figure 5.2.2.5.1-2)
\ean13[scale=SC0, showdigits=false, code=9771384524017, addon=12]

% GS1 example (figure 5.2.2.5.2-2)
\ean13[scale=SC0, showdigits=false, code=9780877799306, addon=54321]

% GS1 example (figure 5.2.6.6-1)
\ean13[scale=SC0, showdigits=false, code=5012345678900]

% GS1 example (figure 5.2.6.6-6)
\ean13[scale=SC0, showdigits=false, code=1234567891019, addon=45678]

\end{document}
