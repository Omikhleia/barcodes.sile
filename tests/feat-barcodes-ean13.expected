Set paper size 	297.6377985	419.5275636
Begin page
Mx 	23.1137
My 	72.7937
Draw line	23.1137	20.9764	0.6917	55.5591
Draw line	24.6104	20.9764	0.6917	55.5591
Draw line	26.1071	20.9764	1.4400	51.8173
Draw line	29.8488	20.9764	0.6917	51.8173
Draw line	31.3455	20.9764	0.6917	51.8173
Draw line	33.5906	20.9764	2.1883	51.8173
Draw line	36.5839	20.9764	1.4400	51.8173
Draw line	39.5773	20.9764	1.4400	51.8173
Draw line	42.5707	20.9764	1.4400	51.8173
Draw line	45.5641	20.9764	0.6917	51.8173
Draw line	47.0608	20.9764	0.6917	51.8173
Draw line	49.3058	20.9764	2.1883	51.8173
Draw line	53.0476	20.9764	1.4400	51.8173
Draw line	56.0409	20.9764	0.6917	51.8173
Draw line	57.5376	20.9764	0.6917	55.5591
Draw line	59.0343	20.9764	0.6917	55.5591
Draw line	60.5310	20.9764	0.6917	51.8173
Draw line	62.7761	20.9764	2.1883	51.8173
Draw line	65.7694	20.9764	0.6917	51.8173
Draw line	69.5112	20.9764	0.6917	51.8173
Draw line	71.0079	20.9764	1.4400	51.8173
Draw line	74.0013	20.9764	1.4400	51.8173
Draw line	76.2463	20.9764	2.1883	51.8173
Draw line	79.9880	20.9764	0.6917	51.8173
Draw line	81.4847	20.9764	2.1883	51.8173
Draw line	85.2265	20.9764	0.6917	51.8173
Draw line	86.7232	20.9764	2.1883	51.8173
Draw line	90.4649	20.9764	0.6917	51.8173
Draw line	91.9616	20.9764	0.6917	55.5591
Draw line	93.4583	20.9764	0.6917	55.5591
My 	132.9357
Draw line	23.1137	81.1184	0.6917	55.5591
Draw line	24.6104	81.1184	0.6917	55.5591
Draw line	26.1071	81.1184	2.1883	51.8173
Draw line	29.1005	81.1184	1.4400	51.8173
Draw line	32.0939	81.1184	0.6917	51.8173
Draw line	35.0872	81.1184	0.6917	51.8173
Draw line	36.5839	81.1184	1.4400	51.8173
Draw line	39.5773	81.1184	1.4400	51.8173
Draw line	41.8224	81.1184	2.9367	51.8173
Draw line	45.5641	81.1184	0.6917	51.8173
Draw line	48.5575	81.1184	0.6917	51.8173
Draw line	50.8025	81.1184	0.6917	51.8173
Draw line	52.2992	81.1184	0.6917	51.8173
Draw line	55.2926	81.1184	1.4400	51.8173
Draw line	57.5376	81.1184	0.6917	55.5591
Draw line	59.0343	81.1184	0.6917	55.5591
Draw line	60.5310	81.1184	0.6917	51.8173
Draw line	62.7761	81.1184	2.1883	51.8173
Draw line	65.7694	81.1184	1.4400	51.8173
Draw line	68.0145	81.1184	1.4400	51.8173
Draw line	71.0079	81.1184	0.6917	51.8173
Draw line	72.5046	81.1184	2.1883	51.8173
Draw line	76.2463	81.1184	2.1883	51.8173
Draw line	79.9880	81.1184	0.6917	51.8173
Draw line	81.4847	81.1184	1.4400	51.8173
Draw line	84.4781	81.1184	1.4400	51.8173
Draw line	86.7232	81.1184	0.6917	51.8173
Draw line	89.7165	81.1184	0.6917	51.8173
Draw line	91.9616	81.1184	0.6917	55.5591
Draw line	93.4583	81.1184	0.6917	55.5591
Mx 	103.1868
Draw line	103.1868	87.0144	0.6917	49.6630
Draw line	104.6835	87.0144	1.4400	49.6630
Draw line	107.6769	87.0144	1.4400	49.6630
Draw line	110.6702	87.0144	0.6917	49.6630
Draw line	112.1669	87.0144	0.6917	49.6630
Draw line	114.4120	87.0144	0.6917	49.6630
Draw line	116.6570	87.0144	1.4400	49.6630
Mx 	23.1137
My 	193.0777
Draw line	23.1137	141.2604	0.6917	55.5591
Draw line	24.6104	141.2604	0.6917	55.5591
Draw line	26.1071	141.2604	2.1883	51.8173
Draw line	29.1005	141.2604	1.4400	51.8173
Draw line	32.8422	141.2604	0.6917	51.8173
Draw line	35.0872	141.2604	0.6917	51.8173
Draw line	36.5839	141.2604	0.6917	51.8173
Draw line	38.8290	141.2604	2.1883	51.8173
Draw line	41.8224	141.2604	1.4400	51.8173
Draw line	44.0674	141.2604	2.1883	51.8173
Draw line	47.8091	141.2604	0.6917	51.8173
Draw line	50.8025	141.2604	0.6917	51.8173
Draw line	52.2992	141.2604	2.1883	51.8173
Draw line	55.2926	141.2604	1.4400	51.8173
Draw line	57.5376	141.2604	0.6917	55.5591
Draw line	59.0343	141.2604	0.6917	55.5591
Draw line	60.5310	141.2604	0.6917	51.8173
Draw line	63.5244	141.2604	0.6917	51.8173
Draw line	65.7694	141.2604	2.1883	51.8173
Draw line	68.7628	141.2604	0.6917	51.8173
Draw line	71.0079	141.2604	2.1883	51.8173
Draw line	74.0013	141.2604	0.6917	51.8173
Draw line	76.2463	141.2604	0.6917	51.8173
Draw line	79.9880	141.2604	0.6917	51.8173
Draw line	81.4847	141.2604	2.1883	51.8173
Draw line	85.2265	141.2604	0.6917	51.8173
Draw line	86.7232	141.2604	0.6917	51.8173
Draw line	88.2198	141.2604	0.6917	51.8173
Draw line	91.9616	141.2604	0.6917	55.5591
Draw line	93.4583	141.2604	0.6917	55.5591
Mx 	103.1868
Draw line	103.1868	147.1564	0.6917	49.6630
Draw line	104.6835	147.1564	1.4400	49.6630
Draw line	106.9285	147.1564	2.1883	49.6630
Draw line	110.6702	147.1564	0.6917	49.6630
Draw line	112.1669	147.1564	0.6917	49.6630
Draw line	113.6636	147.1564	0.6917	49.6630
Draw line	116.6570	147.1564	1.4400	49.6630
Draw line	118.9020	147.1564	0.6917	49.6630
Draw line	120.3987	147.1564	0.6917	49.6630
Draw line	124.1405	147.1564	0.6917	49.6630
Draw line	125.6372	147.1564	0.6917	49.6630
Draw line	127.8822	147.1564	0.6917	49.6630
Draw line	130.1272	147.1564	1.4400	49.6630
Draw line	132.3723	147.1564	0.6917	49.6630
Draw line	134.6173	147.1564	1.4400	49.6630
Draw line	137.6107	147.1564	0.6917	49.6630
Mx 	23.1137
My 	252.8796
Draw line	23.1137	201.0622	0.6917	55.5591
Draw line	24.6104	201.0622	0.6917	55.5591
Draw line	27.6038	201.0622	1.4400	51.8173
Draw line	29.8488	201.0622	0.6917	51.8173
Draw line	31.3455	201.0622	1.4400	51.8173
Draw line	34.3389	201.0622	1.4400	51.8173
Draw line	37.3323	201.0622	1.4400	51.8173
Draw line	39.5773	201.0622	1.4400	51.8173
Draw line	41.8224	201.0622	2.9367	51.8173
Draw line	45.5641	201.0622	0.6917	51.8173
Draw line	47.0608	201.0622	0.6917	51.8173
Draw line	50.0542	201.0622	1.4400	51.8173
Draw line	52.2992	201.0622	2.1883	51.8173
Draw line	56.0409	201.0622	0.6917	51.8173
Draw line	57.5376	201.0622	0.6917	55.5591
Draw line	59.0343	201.0622	0.6917	55.5591
Draw line	60.5310	201.0622	0.6917	51.8173
Draw line	62.0277	201.0622	0.6917	51.8173
Draw line	65.7694	201.0622	0.6917	51.8173
Draw line	68.7628	201.0622	0.6917	51.8173
Draw line	71.0079	201.0622	0.6917	51.8173
Draw line	73.2529	201.0622	0.6917	51.8173
Draw line	76.2463	201.0622	2.1883	51.8173
Draw line	79.2397	201.0622	0.6917	51.8173
Draw line	81.4847	201.0622	2.1883	51.8173
Draw line	85.2265	201.0622	0.6917	51.8173
Draw line	86.7232	201.0622	2.1883	51.8173
Draw line	90.4649	201.0622	0.6917	51.8173
Draw line	91.9616	201.0622	0.6917	55.5591
Draw line	93.4583	201.0622	0.6917	55.5591
My 	313.0216
Draw line	23.1137	261.2042	0.6917	55.5591
Draw line	24.6104	261.2042	0.6917	55.5591
Draw line	26.8554	261.2042	0.6917	51.8173
Draw line	29.1005	261.2042	1.4400	51.8173
Draw line	31.3455	261.2042	2.9367	51.8173
Draw line	35.0872	261.2042	0.6917	51.8173
Draw line	37.3323	261.2042	2.1883	51.8173
Draw line	40.3257	261.2042	0.6917	51.8173
Draw line	41.8224	261.2042	1.4400	51.8173
Draw line	45.5641	261.2042	0.6917	51.8173
Draw line	49.3058	261.2042	0.6917	51.8173
Draw line	50.8025	261.2042	0.6917	51.8173
Draw line	53.0476	261.2042	0.6917	51.8173
Draw line	56.0409	261.2042	0.6917	51.8173
Draw line	57.5376	261.2042	0.6917	55.5591
Draw line	59.0343	261.2042	0.6917	55.5591
Draw line	60.5310	261.2042	0.6917	51.8173
Draw line	62.7761	261.2042	0.6917	51.8173
Draw line	65.7694	261.2042	2.1883	51.8173
Draw line	68.7628	261.2042	0.6917	51.8173
Draw line	71.0079	261.2042	1.4400	51.8173
Draw line	74.0013	261.2042	1.4400	51.8173
Draw line	76.2463	261.2042	2.1883	51.8173
Draw line	79.9880	261.2042	0.6917	51.8173
Draw line	81.4847	261.2042	1.4400	51.8173
Draw line	84.4781	261.2042	1.4400	51.8173
Draw line	86.7232	261.2042	2.1883	51.8173
Draw line	89.7165	261.2042	0.6917	51.8173
Draw line	91.9616	261.2042	0.6917	55.5591
Draw line	93.4583	261.2042	0.6917	55.5591
Mx 	103.1868
Draw line	103.1868	267.1003	0.6917	49.6630
Draw line	104.6835	267.1003	1.4400	49.6630
Draw line	107.6769	267.1003	2.1883	49.6630
Draw line	110.6702	267.1003	0.6917	49.6630
Draw line	112.1669	267.1003	0.6917	49.6630
Draw line	113.6636	267.1003	1.4400	49.6630
Draw line	117.4054	267.1003	0.6917	49.6630
Draw line	118.9020	267.1003	0.6917	49.6630
Draw line	120.3987	267.1003	0.6917	49.6630
Draw line	121.8954	267.1003	2.9367	49.6630
Draw line	125.6372	267.1003	0.6917	49.6630
Draw line	127.8822	267.1003	0.6917	49.6630
Draw line	130.8756	267.1003	0.6917	49.6630
Draw line	132.3723	267.1003	0.6917	49.6630
Draw line	133.8690	267.1003	1.4400	49.6630
Draw line	136.1140	267.1003	2.1883	49.6630
End page
Finish
